ruby/vampires.rb

```diff
@@ -0,0 +1,83 @@
+puts "how many employees will be processed?"
+employee_count = gets.chomp.to_i
+survey_count = 0
+
+while survey_count < employee_count
+survey_count += 1
+
+puts "What is your name?"
+user_name = gets.chomp
+
+
+
+puts "How old are you?"
+stated_age = gets.chomp.to_i
+
+puts "What year were you born?"
+birthYear = gets.chomp.to_i
+
+calculated_age = (2017 - birthYear)
+actual_age = stated_age == calculated_age
+wrong_age = stated_age != calculated_age
+
+puts "do you like garlic? (y/n)"
+input = gets.chomp
+
+
+if input == "y"
+  likes_garlic = true
+else
+  likes_garlic = false
+end
+
+puts "Do you want insurance? (y/n)"
+input = gets.chomp
+
+if input == "y"
+  wants_insurance = true
+else
+  wants_insurance = false
+end
+
+puts "do you have any allergies? please hit return after each allergy listed.  When complete, type: done followed by the return key."
+user_allergy = ""
+while user_allergy != "done"
+user_allergy = gets.chomp
+if user_allergy == "sunshine"
+  result = "probably a vampire"
+  break
```

Comment:
It is always a good idea to try and let our loops break naturally.  Using `break` is an abrupt stop.

A good refactor suggestion here would be be to end your while loop if `"done" || "sunshine"`